### ISSUE TITLE
Cannot use keyboard

### ISSUE THREAD

**Issue by bebop3178** [Luxocracy/NeatPlates#359]
Since I installed the latest version, this addon is causing my game to not respond to any keyboard key. 

The only thing I can do is click in the chat editbox and then I can type stuff, that's it.

Let me know if you need any other info, thanks!

**Comment by Fiveby21** [comment]
I'm also having this issue. I disabled my addons and determined that neatplates was causing the problem.

**Comment by orcphelia** [comment]
Same

**Comment by Luxocracy** [comment]
Neatplates is not yet updated for the Dragon flight prepatch. There is an alpha version that makes it usable, but even that sadly isn't completely working with everything being added in the pre-patch.

Hopefully I can fix the biggest issues with the alpha today.

**Comment by bebop3178** [comment]
> Neatplates is not yet updated for the Dragon flight prepatch. There is an alpha version that makes it usable, but even that sadly isn't completely working with everything being added in the pre-patch.
> 
> Hopefully I can fix the biggest issues with the alpha today.

Thanks! Your work is deeply appreciated. I saw it was updated a few days ago but indeed, it still needs a bit more to be fully compatible.

**Comment by Luxocracy** [comment]
I've released a new non-alpha version of the addon. It is basically the same as the alpha version as the bigger issues I experienced on the PTR, I was not able to replicate on the live servers. So hopefully it was something specific to the PTR and it will continue to work on the liver servers.

There's probably quite some stuff that I haven't yet been able to fix or maybe even notice, so if you do see issues I would appreciate it if you reported them so they can be fixed.
